### PR TITLE
find_ functions for sites, coords

### PR DESCRIPTION
**Pull request by aappling-usgs**

### CODE REVIEW

**Comment by aappling-usgs** [comment]
@lawinslow merged your changes and added some small functions - you have time to review & merge?

File: R/stage_nldas_ts.R [hunk 1]
```diff
@@ -1,55 +1,51 @@
...
-#'@importFrom dataRetrieval readNWISsite
-#'@importFrom unitted u write_unitted
-#'
-#'@examples
-#'\dontrun{
-#'files <- stage_nldas_ts(sites = c("nwis_06893820","nwis_01484680"), variable = "baro", 
-#'                 times = c('2014-01-01 00:00','2014-01-01 05:00'))
-#'}
-#'@export
+#' @title stage ndlas data into a time series file
```
Comment (lawinslow) [inline R/stage_nldas_ts.R]: nldas

File: R/find_site_coords.R [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#' @param format character specifying the desired output format. normal is a 
+#'   human-readable, dplyr-friendly format. geoknife works well for geoknife and
+#'   excludes sites with incomplete coordinate data.
+#' @return a data.frame of
+#' @export
+#' @examples 
+#' # middle site has missing coords:
+#' find_site_coords(c("nwis_01467200","nwis_09327000","nwis_351111089512501")) 
+#' find_site_coords(c("nwis_01467200","nwis_09327000","nwis_351111089512501"), format="geoknife")
+find_site_coords <- function(site_names, format=c("normal","geoknife")) {
```
Comment (lawinslow) [inline R/find_site_coords.R]: now that we're replacing most powstreams functionality, I want to vote once again we merge and drop powstreams

**Comment by lawinslow** [comment]
Looks good!

File: R/find_site_coords.R (hunk 2 above)
Comment (jread-usgs) [inline R/find_site_coords.R]: when was the first vote?

Comment (lawinslow) [inline R/find_site_coords.R]: https://github.com/USGS-R/powstreams/pull/32

Comment (aappling-usgs) [inline R/find_site_coords.R]: at lunch =). observing that all the functionality is moving into mda.streams b/c we need it there. leaves very little for powstreams to do besides wrap or rename functions and protect us/our collaborators from unintentional SB modification while using powstreams. and luke pointed out we could just give collaborators read-only access to SB.

Comment (jread-usgs) [inline R/find_site_coords.R]: yeah, wasn't saying I didn't agree...just figured I missed something.

Comment (aappling-usgs) [inline R/find_site_coords.R]: you sure did miss something! gotta stay on your toes around here - things happen fast =)

**Pull request merged**